packages/aws-client/package.json

```diff
@@ -43,6 +43,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-sdk/client-api-gateway": "^3.58.0",
```

Comment:
We should update all `@aws-sdk/` packages to have the same version.

Reply:
[4cf9990](https://github.com/nasa/cumulus/pull/2915/commits/4cf99900f1eb91290c7f4432c07b06a7a7ba576c)